x-pack/plugins/siem/server/lib/hosts/elasticsearch_adapter.ts

```diff
@@ -74,3 +100,36 @@ export const formatHostsData = (
       },
     }
   );
+
+const formatHostItem = (fields: ReadonlyArray<string>, bucket: HostAggEsItem): HostItem =>
+  fields.reduce<HostItem>((flattenedFields, fieldName) => {
+    const fieldValue = getHostFieldValue(fieldName, bucket);
+    if (fieldValue) {
```

Comment:
You have to be careful about an `if` with a string like this because if this is an empty string then it will not evaluate as true.

```ts
if ("") {
  console.log("You won't see this");
} else {
  console.log("You will see this");
}
You will see this
```
Is that your intention? If not, I would do a `if (fieldValue != null)` instead.